thehive-charts/thehive/README.md

```diff
@@ -1,5 +1,227 @@
+| cassandra.cluster.rack | string | `"Rack1-TheHive"` | Cassandra rack name |
+| cassandra.dbUser.existingSecret | string | `""` | Name of existing secret with Cassandra password (expects .data.cassandra-password key) |
+| cassandra.dbUser.forcePassword | bool | `false` | Force password update on deployment |
+| cassandra.dbUser.password | string | `"ChangeThisPasswordForCassandra"` | Cassandra database password |
+| cassandra.dbUser.user | string | `"cassandra"` | Cassandra database username |
+| cassandra.enabled | bool | `true` | Enable Cassandra dependency subchart deployment |
+| cassandra.extraEnvVars | list | `[{"name":"CASSANDRA_AUTHENTICATOR","value":"PasswordAuthenticator"},{"name":"CASSANDRA_AUTHORIZER","value":"CassandraAuthorizer"}]` | Extra environment variables for Cassandra |
+| cassandra.extraEnvVars[0] | object | `{"name":"CASSANDRA_AUTHENTICATOR","value":"PasswordAuthenticator"}` | Cassandra authenticator (use AllowAllAuthenticator for passwordless) |
+| cassandra.extraEnvVars[1] | object | `{"name":"CASSANDRA_AUTHORIZER","value":"CassandraAuthorizer"}` | Cassandra authorizer (use AllowAllAuthorizer for passwordless) |
+| cassandra.global.security.allowInsecureImages | bool | `true` | Allow insecure images to use bitnamilegacy repository |
+| cassandra.image.registry | string | `"docker.io"` | Cassandra image registry |
+| cassandra.image.repository | string | `"bitnamilegacy/cassandra"` | Cassandra image repository |
+| cassandra.image.tag | string | `"4.1.7-debian-12-r3"` | Cassandra image tag |
+| cassandra.jvm.extraOpts | string | `"-Xms2g -Xmx2g -Xmn200m"` | Additional JVM options for Cassandra |
+| cassandra.jvm.maxHeapSize | string | `"2g"` | Maximum heap size for Cassandra JVM |
+| cassandra.jvm.newHeapSize | string | `"200m"` | New generation heap size for Cassandra JVM |
+| cassandra.networkPolicy.enabled | bool | `false` | Enable network policy for Cassandra |
+| cassandra.pdb.create | bool | `true` | Create PodDisruptionBudget for Cassandra |
+| cassandra.pdb.maxUnavailable | string | `"1"` | Maximum unavailable Cassandra pods |
+| cassandra.persistence.enabled | bool | `true` | Enable persistent volume for Cassandra data |
+| cassandra.persistence.size | string | `"10Gi"` | Size of Cassandra persistent volume |
+| cassandra.persistence.storageClass | string | `""` | Storage class for Cassandra persistent volume |
+| cassandra.replicaCount | int | `1` | Number of Cassandra replicas |
+| cassandra.resources | object | `{"limits":{"cpu":"2000m","ephemeral-storage":"4Gi","memory":"3584Mi"},"requests":{"cpu":"1000m","ephemeral-storage":"4Gi","memory":"2048Mi"}}` | Resource requests and limits for Cassandra pods |
+| dynamicSeedDiscovery.image.registry | string | `"docker.io"` | Dynamic seed discovery image registry |
+| dynamicSeedDiscovery.image.repository | string | `"bitnamilegacy/os-shell"` | Dynamic seed discovery image repository |
+| dynamicSeedDiscovery.image.tag | string | `"12-debian-12-r51"` |  |
```

Comment:
I think that some comments in the `values.yaml` file were deleted when you did a git merge with the squashed commit from PR 115, see the changes in commit https://github.com/StrangeBeeCorp/helm-charts/pull/118/commits/a57957efc28ca0169f9bf9ac17d56246589eba2a